jvb/src/main/java/org/jitsi/videobridge/Videobridge.java

```diff
@@ -598,7 +630,7 @@ public void stop()
     public OrderedJsonObject getDebugState(String conferenceId, String endpointId, boolean full)
     {
         OrderedJsonObject debugState = new OrderedJsonObject();
-        debugState.put("shutdownInProgress", shutdownInProgress);
+        debugState.put("shutdownInProgress", (shutdownRequestedTime != null));
```

Comment:
Do we want to have the actual requested time in the debug state?

Reply:
I think having it in the logs is sufficient.